doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # We will collect the 5 best answers.
</pre><p>move ths line to the code block below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>That won't work unfortunately. The %% magic is a bit fussy about what can be in the cell.</p>

Reply:
<p>well then remove it and add a text section between the two code blocks instead?</p>